Update OCP-4-13-release-notes for Developer Perspective: New Features (#57916)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-web-console"]
=== Web console
[id="ocp-4-13-developer-perspective"]
==== Developer Perspective

> With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:

Author: Made this change. Thanks!

> * Create a *Serverless Function* through the *Import from Git* flow.

Reviewer: ```suggestion
* Create a *Serverless Function* by using the *Import from Git* flow.
```

Author: Noted and made this change. Thanks!

> * Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.

Reviewer: ```suggestion
* Create a *Serverless Function* by using the *Create Serverless Function* flow available on *Add page*.
```

Author: Noted and made this change. Thanks!

> * The *Import from Git* workflow now supports *pipeline-as-code*.

Reviewer: ```suggestion
* Use *pipeline-as-code* as an option in the *Import from Git* workflow.
```
The list item needs to have parallel syntactic structure with the other list items. My edit is one way to achieve this change, but feel free to try alternatives if my suggestion doesn't feel right to you.

Author: Made this change. Thanks!

> * See which pods are receiving traffic in the:

Reviewer: ```suggestion
* See which pods are receiving traffic in the following places:
```
See here for guidance regarding lead-in wording for lists.

> ** Side pane of the *Topology* view

Reviewer: ```suggestion
** The side pane of the *Topology* view
```
Add definitel article

Author: Made this change. Thanks!

> ** Pod *Details* view

Reviewer: ```suggestion
** The *Details* view for a pod
```

Author: Made this change. Thanks!

> ** *Pods* list view

Author: Made this change. Thanks!

* Customize the timeout period or provide your own image when instantiating a *Web Terminal*.

> * Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all the users.

Reviewer: ```suggestion
* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all users.
```
Remove definite article

Author: Made this change. Thanks!

[id="ocp-4-13-pipelines-page-improvements"]
===== Pipelines page improvements
In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:

> * The previously selected tab is visible when returning to the *Pipelines* page.

Reviewer: ```suggestion
* The tab you previously selected remains visible when you return to the *Pipelines* page.
```
- A couple of suggestions for improving user-centric wording

Author: Made this change. Thanks!

> * The default tab for *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.

Reviewer: ```suggestion
* The default tab for the *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
```
add definite article

[id="ocp-4-13-helm-page-improvements"]

> ===== Helm page improvements

Author: If you look at our documentation, we have not used Helm as an attribute anywhere. I'm skeptical to change this. Skipping this suggestion for now.

> ===== Helm page improvements

Reviewer: Ok. Understood. Thanks!

> In {product-title} 4.13, you can do the following from the *Helm* page:

Reviewer: ```suggestion
In {product-title} 4.13, you can now do the following from the *Helm* page:
```

Author: This looks good. I'm using your version and making the change.

> 

Reviewer: The following list needs to be re-worded to make each item grammatically parallel: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items

Author: I have followed your suggestions mentioned for the list and the lead-in sentence.

* New terminology: Create and delete Helm releases instead of installing and uninstalling Helm charts.

> * Create and delete helm releases asynchronously and not wait for these actions to complete before performing the next task in the web console.

Reviewer: ```suggestion
* You can create and delete Helm releases asynchronously and not wait for actions to complete before performing the next task in the web console.
```
I would suggest first updating the lead-in sentence, and then rewording this item to something similar to my suggestion.

Author: Made this change and as mentioned above I'm skipping the second suggestion.

> * Helm release list now includes a *Status* column.

Reviewer: ```suggestion
* The Helm release list now includes a *Status* column.
```
- Add definite article.
- Should `Helm` use an attribute?

Author: Made the first change and skipping the second suggestion.
